### ISSUE TITLE
sci-biology/betse-1.1.1.ebuild impossible use flags

### ISSUE THREAD

**Issue by impiusnex** [leycec/raiagent#77]
Greetings,

I'm trying to build sci-biology/betsee-1.1.1.0, and I find myself unable to do so as there is a useflag dependency that can not be met.
The error message I receive while building is 
```
emerge: there are no ebuilds built with USE flags to satisfy "sci-libs/atlas[eselect-ldso,threads]".
!!! One of the following packages is required to complete your request:
- sci-libs/atlas-3.10.2::gentoo (Missing IUSE: eselect-ldso)
(dependency required by "sci-biology/betse-1.1.1::raiagent[smp]" [ebuild])
(dependency required by "sci-biology/betsee-1.1.1.0::raiagent" [ebuild])
(dependency required by "betsee" [argument])
```

The ebuild for sci-libs/atlas-3.10.2 does not have the 'eselect-ldso' use flag.

**Comment by leycec** [comment]
I... *I can't believe it.* Someone is actually attempting to install [BETSE](https://gitlab.com/betse/betse) under Gentoo. Admittedly, they're failing due to my own ineptitude. But, still – this is a special day for me that will go down in personal infamy.

That said, I'm deeply saddened to learn that `sci-libs/atlas` no longer supports [BLAS/LAPACK runtime switching](https://wiki.gentoo.org/wiki/Blas-lapack-switch). That renders ATLAS effectively useless for real-world scientific taskloads under Gentoo, which is less than optimal. On the bright side, I suppose everyone's probably already migrated to BLIS and OpenBLAS anyway.

I'll promptly remove all mention of `sci-libs/atlas` from BETSE ebuilds. Thanks for the heads up, @impiusnex. Love the guru beard, too! And a fellow Cannuck at that. *We are legion. And very cold.*

**Comment by leycec** [comment]
**Resolved by c0f451e.** :fireworks:
